compare-master-to-stable.js

```diff
@@ -9,72 +9,72 @@ var Q = require('q');
 var _ = require('lodash');
 var semver = require('semver');
 
-var exec = function (cmd) {
-  return function () {
+var exec = function(cmd) {
+  return function() {
     var args = Array.prototype.slice.call(arguments, 0);
     args.unshift(cmd);
     var fullCmd = util.format.apply(util, args);
-    return Q.nfcall(cp.exec, fullCmd).then(function (out) {
+    return Q.nfcall(cp.exec, fullCmd).then(function(out) {
       return out[0].split('\n');
     });
   };
 };
 
-var andThen = function (fn, after) {
-  return function () {
+var andThen = function(fn, after) {
+  return /* @this */ function() {
     return fn.apply(this, arguments).then(after);
   };
 };
 
-var oneArg = function (fn) {
-  return function (arg) {
+var oneArg = function(fn) {
+  return function(arg) {
     return fn(arg);
   };
 };
 
-var oneLine = function (lines) {
+var oneLine = function(lines) {
   return lines[0].trim();
 };
 
-var noArgs = function (fn) {
-  return function () {
+var noArgs = function(fn) {
+  return function() {
     return fn();
   };
 };
 
-var identity = function (i) { return i; };
+var identity = function(i) { return i; };
 
 // like Q.all, but runs the commands in series
 // useful for ensuring env state (like which branch is checked out)
-var allInSeries = function (fn) {
-  return function (args) {
+var allInSeries = function(fn) {
+  return function(args) {
     var results = [];
     var def;
     while (args.length > 0) {
-      (function (arg) {
+      (function(arg) {
         if (def) {
-          def = def.then(function () {
+          def = def.then(function() {
             return fn(arg);
           });
         } else {
           def = fn(arg);
         }
-        def = def.then(function (res) {
+        def = def.then(function(res) {
           results.push(res);
         });
-      }(args.pop()));
+      })(args.pop());
```

Comment:
Aha, I didn't know you need to choose one or the other with ESLInt.
(I prefer `outside`, but I think it is a personal preference. `Inside` is fine.)